src/lib/web-normalised-data-info.js

```diff
@@ -0,0 +1,112 @@
+import { database_pool } from './database.js';
+
+class  NormalisedDataInfo {
+  static async getInfo(data_id) {
+
+    const normalised_data_sql_query = `
+     SELECT
+        *
+      FROM normalised_data
+      WHERE data_id=$1 LIMIT 1
+    `;
+
+    const client = await database_pool.connect();
+
+    let out;
+
+    try {
+      const res_normalised_data = await client.query(normalised_data_sql_query, [data_id]);
+      if (res_normalised_data.rows.length == 1){
+        const normalised_data = res_normalised_data.rows[0];
+        out = {
+            'id': normalised_data.data_id,
+            'deleted': normalised_data.data_deleted,
+            'data': normalised_data.data,
+            'updated_at': normalised_data.updated_at,
+            'kind': normalised_data.data_kind
+        };
+
+        out['raw_data'] = await this.get_raw_data_info(normalised_data.raw_data_id, client);
+
```

Comment:
normally in JS you'd just do `out.raw_data = ...etc`  you generally only use the assoc array stuff if you have a variable as a name e.g. `let key = "moo"; out[key] = ...etc`